lib/functions/lintly.sh

```diff
@@ -35,44 +35,42 @@ function AddLinterOptsForLintly() {
   [[ ! "${LINTER_OPTS[DOCKERFILE_HADOLINT]}" =~ "json" ]] && LINTER_OPTS[DOCKERFILE_HADOLINT]+=" --format json"
   [[ ! "${LINTER_OPTS[TERRAFORM_TERRASCAN]}" =~ "json" ]] && LINTER_OPTS[TERRAFORM_TERRASCAN]+=" -o json"
   [[ ! "${LINTER_OPTS[CLOUDFORMATION_CFN_NAG]}" =~ "json" ]] && LINTER_OPTS[CLOUDFORMATION_CFN_NAG]+=" --output-format=json"
-  # gitleaks with either --verbose or --quiet will write extra logs to stdout and mess up our report; remove these
```

Comment:
This here is okay to remove now; for gitleaks we used to send the report file to `/dev/stdout`, so we had to make sure nothing *else* was written to stdout (which both `--quiet` and `--verbose` were doing).

Now we're having `--report` directly send the report (the file we actually want!) to our temp file, so we can remove this (as well as add `--verbose` to `LINTER_COMMANDS_ARRAY['GITLEAKS']` below).